merlin/systems/triton/conversions.py

```diff
@@ -321,14 +317,9 @@ def _cudf_to_array(df, cpu=True):
     for name in df.columns:
         col = df[name]
         if is_list_dtype(col.dtype):
-            values = col.list.leaves.values_host if cpu else col.list.leaves.values
             offsets = col._column.offsets.values_host if cpu else col._column.offsets.values
-
-            row_lengths = offsets[1:] - offsets[:-1]
-            if all(row_lengths == row_lengths[0]):
-                output[name] = values.reshape((-1, row_lengths[0]))
-            else:
-                output[name] = (values, offsets)
+            values = col.list.leaves.values_host if cpu else col.list.leaves.values
+            output[name] = (values, offsets)
```

Comment:
This actively rolls back some of the changes that made the T4R notebook work. I don't think we should use values/offsets for all list features, just the ragged ones.

Reply:
It seemed easier to do it this way, rather than trying to conditionally convert from a fixed size tensor back into the ragged represntation in the case where we have one row or happen to have rows of the same length (but still want to output the ragged representation to match the workflow output schema and the corresponding triton model